src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import './Tabs.scss';
+
+export const Tabs = ({ tabs, selectedTabId, selectTab }) => (
+  <div className="Tabs">
+    <div className="Tabs__header">
+      {
+  tabs.map(({ title, id }) => (
+    <button
+      key={id}
+      type="button"
+      className={classNames('Tabs__title', {
+        'Tabs__title-selected': +id.slice(4) === selectedTabId,
```

Comment:
I think, it will be better, if you change class name from 'Tabs__title-selected' to 'Tabs__selected-title'.